### ISSUE TITLE
[en] Missing 'had' in 'had better' not found

### ISSUE THREAD

**Issue by MikeUnwalla** [languagetool-org/languagetool#4311]
Not found, missing 'had' before 'better': 

- [ ] That better work well, or I will be mad.
- [ ] Jim better be quick.
- [ ] Marketing better show some respect.
- [ ] That better become large.
- [ ] This better be important. (Was found before improvement https://github.com/languagetool-org/languagetool/commit/b4112838bfd0be2ae8e58babf375fb3cdb8d2cb3)

Reference: https://www.lexico.com/definition/better

**Comment by udomai** [comment]
@hftf — do you agree? Is it any more formal/correct in AE with the "had"? Our judgement might be too cis-Atlantic ;)

**Comment by hftf** [comment]
@udomai I agree, with the caveat that, as according to the Usage note in the Lexico entry, it should be indicated as informal rather than incorrect, and `’d` should probably be suggested along with `had` when it is euphonic (e.g. `You’d` or `He’d`).
